integration/pkg/clnode/start_committee_verifier.go

```diff
@@ -0,0 +1,82 @@
+package clnode
+
+import (
+	"context"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink-ccv/integration/pkg/sourcereader"
+	"github.com/smartcontractkit/chainlink-ccv/protocol"
+	"github.com/smartcontractkit/chainlink-ccv/verifier"
+	"github.com/smartcontractkit/chainlink-ccv/verifier/commit"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-evm/pkg/chains/legacyevm"
+
+	vcommon "github.com/smartcontractkit/chainlink-ccv/verifier/pkg/common"
+)
+
+// StartCCVComitteeVerifier starts the Committee Verifier with evm chains.
+func StartCCVComitteeVerifier(
+	ctx context.Context,
+	lggr logger.Logger,
+	cfg CCVConfig,
```

Comment:
I believe there is already a config struct defined by verifier - can that be used or is it not possible / doesn't have the required info?